common.go

```diff
@@ -52,6 +52,14 @@ func retryWithLimitedNumOfRetries(targetFunc retryableFunc, numOfRetries int, de
 
```

Comment:
Maybe shorter function name? `retryWith` or `retryNTimes` or so instead of `retryWithLimitedNumOfRetries`.

Reply:
Sounds good! Let me change its name.